docs/references/astro/organization-store.mdx

```diff
@@ -40,7 +40,7 @@ export default function Home() {
 
 The following example demonstrates how to implement pagination for organization memberships. The `memberships` state will be populated with the first page of the organization's memberships. When the "Previous page" or "Next page" button is clicked, the `fetchMemberships` function will be called to fetch the previous or next page of memberships.
 
-You can implement this pattern to any Clerk function that returns a [`ClerkPaginatedResponse`](/docs/references/javascript/types/clerk-paginated-response#clerk-paginated-response) object.
+You can implement this pattern to any Clerk function that returns a [`ClerkPaginatedResponse`](https://clerk.com/docs/references/javascript/types/clerk-paginated-response#clerk-paginated-response) object.
```

Comment:
❓ What's the reason behind this change ? In order for me to know when i should use absolute vs relative

Reply:
we discussed it in the groupchat, and it's also written in the description of the pr: "We're going to use external links for links that take you from the Astro SDK to another SDK (such as our Nextjs SDK). This is a temporary solution until we figure out how to deal with related/duped docs."